dev-docs/howtoguides/use_staging_environment.md

```diff
@@ -0,0 +1,10 @@
+# Use the contracts staging environment
+
+If needed, we can change the pro client to use the staging environment
+instead of the production environment. To do that we need to change
+the `contract_url` on the `/etc/ubuntu-advantage/uaclient.conf`. Change
+the value of that key to `https://contracts.staging.canonical.com`.
```

Comment:
I slightly prefer the way you wrote a similar suggestion down in the auto attach reference.

```suggestion
You can change the contract server that UA Client will use by setting the
following option in your `uaclient.conf` configuration file, (by default located at
`/etc/ubuntu-advantage/uaclient.conf`).

`https://contracts.staging.canonical.com` is the staging environment of the
contract server

```yaml
contract_url: https://contracts.staging.canonical.com
```